products/cloudidentity/api.yaml

```diff
@@ -222,6 +143,7 @@ objects:
         input: true
         description: |
           EntityKey of the member.
+        min_version: beta
```

Comment:
How does `exactly_one_of` work when one field is beta and one isn't?

Reply:
It looks like [exactly_one_of is converted directly into terraform](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/templates/terraform/schema_property.erb#L178), but using [get_property_schema_path](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/provider/terraform.rb#L129) and `array.compact`, which seems to be [eliminating the beta field from the resulting golang code](https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-4211-old..auto-pr-4211#diff-8774cce34c1912529ae270958b564cb43298e2afc52e4b7e44b578d96a3b70faR106).

In other words, in GA, the go code ends up with a "exactly one of" constraint that doesn't include beta fields - in this case meaning it only contains one property, at least one of which much be specified, which I believe is the correct behavior.